<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>331</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QWidget" name="widget" native="true">
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>40</y>
          <width>225</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Customer Demographics Distribution</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton3">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>140</y>
          <width>113</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Show Chart</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton2">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>90</y>
          <width>113</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Show Chart</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>190</y>
          <width>191</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Promo Code Usage by Gender</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>140</y>
          <width>181</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Sales by Hour of the Day </string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>95</y>
          <width>201</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Sales Trends by Day of the Week</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton4">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>190</y>
          <width>113</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Show Chart</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton1">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>40</y>
          <width>113</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Show Chart</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>260</y>
          <width>113</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Copyright 2023.12</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
